pandas/tests/indexing/test_timedelta.py

```diff
@@ -40,3 +39,13 @@ def test_list_like_indexing(self, indexer, expected):
                                 dtype="int64")
 
         tm.assert_frame_equal(expected, df)
+
+    def test_string_indexing(self):
+        # GH 16896
+        df = pd.DataFrame({'x': range(3)},
+                          index=pd.to_timedelta(range(3), unit='days'))
+        expected = df.iloc[0]
+        sliced = df.loc['0 days']
+        tm.assert_series_equal(sliced, expected)
+
+        assert df.index.get_loc('0 days') == 0
```

Comment:
the .get_loc tests are *already* in ``pandas/tests/indexes/test_base.py`` (which iterates over all index types). Can you see why the .get_loc is NOT hitting this?

you can add some additional tests in ``pandas/tests/indexes/timedeltas/test_indexing.py`` for ``.get_loc`` (just the last line where you are testing vs the index itself).